### PR TITLE
Fix travis and failing tests

### PR DESCRIPTION
**Pull request by oldergod**
:volkswaven:

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ before_script:
   - if [ ! -z "$(git status --porcelain)" ]; then git diff; echo -e "\nTest files changed. Did you run ./gradlew generateTests?"; exit 1; fi
 
 script:
-  - cd wire-library; ../gradlew build; cd ..
+  - cd wire-library && ../gradlew build && cd ..
```
Comment (JakeWharton) [inline .travis.yml]: You can also run this in a subshell with `$(cd wire && ../gradlew build)`

Comment (jrodbx) [reply .travis.yml]: why not `./gradlew -p wire-library`?

Comment (oldergod) [reply .travis.yml]: looks like I could do something like
```yaml
script:
  - ./gradlew -p wire-library build
  - ./gradlew -p . build
```

File: wire-library/wire-kotlin-generator/src/test/java/com/squareup/wire/kotlin/KotlinGeneratorTest.kt [hunk 2]
```diff
@@ -110,22 +110,24 @@ class KotlinGeneratorTest {
...
-          |    path = "/routeguide.RouteGuide/GetFeature",
-          |    requestAdapter = "routeguide.Point#ADAPTER",
-          |    responseAdapter = "routeguide.Feature#ADAPTER"
-          |  )
-          |  fun GetFeature(): GrpcCall<Point, Feature>
+          |  fun GetFeature(): GrpcCall<Point, Feature> = client.newCall(GrpcMethod(
+          |      path = "/routeguide.RouteGuide/GetFeature",
+          |      requestAdapter = Point.ADAPTER,
+          |      responseAdapter = Feature.ADAPTER
+          |  ))
```
Comment (oldergod) [inline wire-library/wire-kotlin-generator/src/test/java/com/squareup/wire/kotlin/KotlinGeneratorTest.kt]: @Egorand I'd rather have something like
```kotlin
fun GetFeature(): GrpcCall<Point, Feature> = client.newCall(
   GrpcMethod(
     path = "/routeguide.RouteGuide/GetFeature",
     requestAdapter = Point.ADAPTER,
     responseAdapter = Feature.ADAPTER
  )
)
```
Is KotlinPoet doing it on purpose? or is it us?

**Pull request merged**